bin/verilator

```diff
@@ -334,6 +337,7 @@ detailed descriptions in L</"VERILATION ARGUMENTS"> for more information.
      -O<optimization-letter>    Selectable optimizations
      -o <executable>            Name of final executable
     --no-order-clock-delay      Disable ordering clock enable assignments
+    --no-verilate               Skip verilation and just compile previously verilated code.
```

Comment:
Need a test_regress test for no-verilate.

Reply:
t_flag_verilate.pl tests both --verilate and --no-verilate.
https://github.com/verilator/verilator/blob/576c4a83401dd70fdaa7a0ff4f540eebddf06a3e/test_regress/t/t_flag_verilate.pl